### DISCUSSION TITLE
0.9.8 - New French Language and New Download/Install Options

### DISCUSSION THREAD

**Discussion by Timthreetwelve** [Timthreetwelve/GetMyIP#57] [Announcements]
### This update includes the French language and new download/installation options 

Apart from bug fixes and translation updates, I expect the pace of updates to Get My IP to slow down as I turn my attention to projects that have been neglected for too long. When I return to Get My IP in 2024, I will be upgrading it to .NET 8, which was released in November and is the next LTS version of .NET.

#### Changes:
- New: French language translation added courtesy of @logounet. 🎉
- New: Added option to hide Exit in the navigation menu. #55
- Updated: NLog and H.NotifyIcon.Wpf dependencies have been updated to their latest versions.
- Updated: Improved resiliency when internet connection is not available.
- New: Additional download and installation options. See below. 

### New download and installation options ✨
With this release I'm adding *Self Contained* versions both the portable and installer versions. In this context, Self Contained means that the required .NET files are included. A self contained installation does **not** require that the .NET runtime is installed. Of course, by including the required .NET files, the installation is much larger. Download the installer or zip archive that best suits your needs.

In addition:
* Publishing a Self Contained build requires that either x86 or x64 be targeted. Consequently, there are versions for each x86 and x64. 
* The filename of the Self Contained builds will indicate the architecture, either `x86` or `x64`.
* The name of the zip archives has been changed from `NonInstall` has been changed to `Portable`.
* The **Checksums** table below has been updated to include a description for each file.

#### Thanks:
My sincere thanks to @logounet for the French translation. 🙏🏻

If you wish to contribute a translation, check out the [Contribute a Translation](https://github.com/Timthreetwelve/GetMyIP/wiki/Contribute-a-Translation) topic in the Wiki. Your help would be awesome.

**Full Changelog**: https://github.com/Timthreetwelve/GetMyIP/compare/v0.9.7...v0.9.8

#### Checksums :
|File Name|Description|SHA256 Hash|
|-|-|-|
|GetMyIP_0.9.8.0__Setup.exe|Installer - Requires .NET 6| `8f4958cbeed5f2946d56b3d19be23c5ddd50d52058c9ea06838c0daf9c2002bc` |
|GetMyIP_0.9.8.0_SC_x86_Setup.exe|Installer - Self Contained x86| `ca6c89961052db16a393a57f9359e95b5ab2fb5afa40c46bfcdba3c067e28740` |
|GetMyIP_0.9.8.0_SC_x64_Setup.exe|Installer - Self Contained x64| `371b408ba03e1a7be6935c2fedc323d8aedfb91faa64b642869265dbe006e867` |
|GetMyIP_0.9.8.0__Portable.zip|Zip Archive - Requires .NET 6| `06af11efc069e3ded00f04b29b25ee89b0875bf830639417e068a0f7815dc0a1` |
|GetMyIP_0.9.8.0_SC_x86_Portable.zip|Zip Archive - Self Contained x86| `98f7178c01b0cd158d8edff73052994ed7942250e04476d89a6737003c0b6cc5` |
|GetMyIP_0.9.8.0_SC_x64_Portable.zip|Zip Archive - Self Contained x64| `b0aecd224b80fdc1e88a15facb205cb111373773cd29ff05b37725eb58cbdb45` |

<hr /><em>This discussion was created from the release <a href='https://github.com/Timthreetwelve/GetMyIP/releases/tag/v0.9.8'>0.9.8 - New French Language and New Download/Install Options</a>.</em>

**Comment by CMTriX** [comment]
Dutch language file updated to 0.9.8
https://github.com/Timthreetwelve/GetMyIP/blob/main/GetMyIP/Languages/Strings.nl-NL.xaml
